index.html

```diff
@@ -208,6 +224,29 @@ <h2> Deliverables </h2>
         <div id="normative">
           <h3> Normative Specifications </h3>
           <dl class='todo'>
+            <dt class="spec"><a href="https://www.w3.org/TR/ttml3/">Timed Text
+                Markup Language 3 (TTML3)<br>
+              </a></dt>
+            <dd>This specification defines a content type that represents timed
+              text media for the purpose of interchange among authoring,
+              distribution and playback systems. Timed text is textual
+              information that is intrinsically or extrinsically associated with
+              timing information. </dd>
```

Comment:
This may consist of one or more documents. Can be not-necessarily TTML3, leave to WG to decide.

Reply:
Done.